package.json

```diff
@@ -107,6 +107,7 @@
         "react-scrollspy": "^3.3.5",
         "react-select": "^3.0.8",
         "react-stickup": "^1.11.0",
+        "react-toastify": "^5.5.0",
```

Comment:
Can you remove this?

Reply:
This should be there. Because, we will use this as our Notification Engine